org.mwc.cmap.legacy/src/MWC/GUI/Properties/PlainPropertyEditor.java

```diff
@@ -1091,7 +1091,7 @@ private void update(final PropertyDescriptor pd, final Object data) {
 
 		// find out if the value for this parameter is different to the
 		// current one for this object - or if there isn't a current value
-		if ((newVal == null) || (!newVal.equals(oldVal))) {
+		if (newVal != oldVal && ((newVal == null) || ( !newVal.equals(oldVal)))) {

```

Comment:
@saulhidalgoaular  - is there a chance of a null-pointer exception here?

It _looks like_ a null newVal could get passed into the right-hand clause.